src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/MarkupCompiler/MarkupCompiler.cs

```diff
@@ -50,7 +50,7 @@ internal sealed class MarkupCompiler
         public string TargetPath
         {
             get { return _targetPath; }
-            set { _targetPath = value; }
+            set { _targetPath = value;}
```

Comment:
Format issues

Reply:
@lindexi: Thank you again for the review.  These formatting changes will be taken care of with the other half of @Nirmal4g's PR.  Thanks.